test/mlir-cpu-runner/unranked_memref.mlir

```diff
@@ -0,0 +1,43 @@
+// RUN: mlir-opt %s -linalg-lower-to-loops -convert-linalg-to-llvm -convert-std-to-llvm | mlir-cpu-runner -e main -entry-point-result=void -shared-libs=%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libcblas%shlibext,%linalg_test_lib_dir/libcblas_interface%shlibext | FileCheck %s
```

Comment:
Can we update the original utils test and adopt the unranked version everywhere?